test/axes/timeAxisTests.ts

```diff
@@ -149,6 +149,60 @@ describe("TimeAxis", () => {
     svg.remove();
   });
 
+  it("tick labels do not overlap with tick marks in top orientation", () => {
+     let svg = TestMethods.generateSVG(400, 100);
+      scale = new Plottable.Scales.Time();
+      scale.domain([new Date("2010-01-01"), new Date("2014-01-01")]);
```

Comment:
Not a big deal, but I would prefer dates to be formatted as
```typescript
new Date(2010, 00, 01);
new Date(2014, 00, 01);
```

But up to you, as we don't have a standard about this yet